test/fixtures/cookbooks/mingw_tdm_install/recipes/install64.rb

```diff
@@ -16,9 +16,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include_recipe 'seven_zip::default'
+include_recipe 'mingw::default'
 
-root_path = node['mingw']['path64']
+root_path = "#{ENV['SYSTEMDRIVE']}\\mingw64"
```

Comment:
We could make the `root` attribute of the `mingw_tdm_gcc` resource support a nice default which is based on the  attribute `flavor`. Take a look at what we did in the languages cookbook:
https://github.com/chef-cookbooks/languages/blob/master/libraries/_prefix.rb
https://github.com/chef-cookbooks/languages/blob/master/libraries/language_install.rb#L28
https://github.com/chef-cookbooks/languages/blob/master/libraries/language_execute.rb#L29